iroha/src/sumeragi.rs

```diff
@@ -2095,7 +2053,7 @@ mod tests {
                 &config.queue_configuration,
             )));
             let mut torii = Torii::from_configuration(
-                &config.torii_configuration,
+                Arc::new(RwLock::new(config.clone())),
```

Comment:
Is it expected behaviour to have different copies of configuration between actors? If true it doesn't require Arc, isn't it?

Reply:
It is just a mock here for now. Forgot to put TODO.

It might be shared between threads in future as it might be solution for changing configuration

Reply:
Added an rfc page for future design:
https://wiki.hyperledger.org/display/iroha/Iroha+configuration+changing